### DISCUSSION TITLE
Unable to clone in local machine

### DISCUSSION THREAD

**Discussion by anjalisingh23** [cockpit-project/cockpit#18402] [Q&A] [answered]
![Screenshot from 2023-02-26 02-08-16](https://user-images.githubusercontent.com/89786958/221378592-92635df3-cf28-4ba8-adee-ea1e35d94bff.png)

**Comment by Prince2319** [comment]
You should Directly run $ git clone https://github.com/cockpit-project/cockpit this then do cd Cockpit

**Comment by martinpitt** [comment, marked as answer]
This suggests an unstable network connection. Please just try again. The command is correct, it should clone your fork at https://github.com/anjalisingh23/cockpit
